index.js

```diff
@@ -27,7 +27,7 @@ CoffeeScriptFilter.prototype.baseDir = function() {
 
 CoffeeScriptFilter.prototype.optionsHash = function() {
   if (!this._optionsHash) {
-    this._optionsHash = stringify(this.options);
```

Comment:
@stefanpenner Would you mind being a bit more clear? Are you suggesting this isn't a change you would want to merge into `broccoli-coffee`? If so, this pull request should probably be closed.